src/qt/rpcconsole.cpp

```diff
@@ -776,20 +776,15 @@ void RPCConsole::setFontSize(int newSize)
 
     // clear console (reset icon sizes, default stylesheet) and re-add the content
     float oldPosFactor = 1.0 / ui->messagesWidget->verticalScrollBar()->maximum() * ui->messagesWidget->verticalScrollBar()->value();
-    clear(false);
+    clear(/* keep_prompt */ true);
```

Comment:
Offtopic: I feel it would be better to use enums, avoiding `/* comments explaining what are we dogin */` I've seen here and there. `clear(CearMode::KeePrompt)` would be self-docummenting, less error prone. But again, that's offtopic for this PR.

Reply:
The relevant discussion is here https://github.com/bitcoin/bitcoin/issues/21684.